build.gradle

```diff
@@ -117,37 +123,31 @@ jar {
                 "Implementation-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ")
         ])
     }
-}
 
-task sourcesJar(type: Jar) {
-    archiveClassifier.set('sources')
-    from sourceSets.main.allJava
+    // This is the preferred method to reobfuscate your jar file
+    finalizedBy 'reobfJar'
 }
 
-artifacts {
-    archives sourcesJar
+tasks.named(sourceSets.main.compileJavaTaskName) {
+    outputs.file("${it.temporaryDir}/${it.name}-refmap.json").withPropertyName("mixin refmap")
 }
 
-jar.finalizedBy('reobfJar')
-
-def reobfFile = file("$buildDir/reobfJar/output.jar")
-def reobfArtifact = artifacts.add('default', reobfFile) {
-    type 'jar'
-    builtBy 'reobfJar'
+compileJava {
+    options.incremental = false
 }
 
 publishing {
     publications {
         mavenJava(MavenPublication) {
             artifactId project.archivesBaseName
             artifact jar
-            artifact sourcesJar
+            artifact sourceJar
         }
     }
     repositories {
         maven {
             name = "GitHubPackages"
-            url = "https://maven.pkg.github.com/The-Aether-Team/Nitrogen"
+            url = "https://maven.pkg.github.com/The-Aether-Team/The-Aether"
```

Comment:
should reference Nitrogen not The-Aether

Reply:
![](https://media.tenor.com/8pU5fB6t260AAAAd/skinny-dance.gif)